lib/src/main/java/co/nimblehq/alpine/lib/mrz/MrzProcessorImpl.kt

```diff
@@ -59,7 +59,8 @@ internal class MrzProcessorImpl : MrzProcessor {
             textBlocks.flatMap { textBlock ->
                 textBlock.lines.flatMap { it.elements }
             }.forEach {
-                filterScannedText(it)
+                if (!isMrzDetected) filterScannedText(it)
+                else return@forEach
```

Comment:
Can write it in one line.
```suggestion
                if (!isMrzDetected) filterScannedText(it) else return@forEach
```